openshift/kustomize/tekton/base/tasks/git-conditions.yaml

```diff
@@ -72,7 +72,7 @@ spec:
           buildVar="${1/\-/\_}"
           # convert to uppercase
           buildVar="${buildVar^^}"
-          BUILD_${buildVar}=true
+          BUILD=BUILD_${buildVar}=true
```

Comment:
Fixed bug I introduced.  However, the test pipeline still doesn't work for some reason.